app/src/exporters/ffmpeg.ts

```diff
@@ -74,3 +81,26 @@ export async function exportContent(
   await combineParts(files, output_path, tempdir);
   fs.rmdirSync(tempdir, { recursive: true });
 }
+
+export async function convertToWav(input_path: string): Promise<Buffer> {
```

Comment:
in the future, we probably want to interleave this with the transcription but for now it seems fine